config_app/js/config-field-templates/config-string-field.html

```diff
@@ -2,7 +2,8 @@
   <form name="fieldform" novalidate>
     <input type="text" class="form-control" placeholder="{{ placeholder || '' }}"
            ng-model="binding" ng-trim="false" ng-minlength="1"
-           ng-pattern="getRegexp(pattern)" ng-required="!isOptional">
+           ng-pattern="getRegexp(pattern)" ng-required="!isOptional"
+           ng-readonly="isReadonly">
```

Comment:
Use [one-time binding](https://docs.angularjs.org/guide/expression#one-time-binding) to improve performance (`::isFieldReadonly(...)`).

Reply:
@alecmerdler Would that work here as well?

Reply:
Yes

Reply:
Fixed